src/components/chat/chatHistory.tsx

```diff
@@ -5,6 +5,8 @@ import { getChats } from '@/services/historyService';
 const ChatHistory: React.FC = async () => {
   const chats = await getChats();
 
+  if (!chats) return null;
```

Comment:
I don't like the idea of chats being able to return null.  I'm not sure what the best pattern would be though.  In dotnet the backend call would just throw an exception because session is required for calling Chats`getChats()`.  So I guess I'd suggest (a) in the `ChatHistory()` component we only call `getChats()` when we have a verified user session, OR `getChats()` w/o a session returns `[]`.

Reply:
i did [] at first, but the i figured if they were logged in but didn't have any chats we would still want to display the empty component